requirements/albu.txt

```diff
@@ -0,0 +1 @@
+albumentations>=0.3.2 --no-binary qudida,albumentations
```

Comment:
I think it's better to move the requirement to [optional.txt](https://github.com/open-mmlab/mmsegmentation/blob/master/requirements/optional.txt)

Reply:
Changed this, but optional requirements seem to be only installed for windows build